src/app.jsx

```diff
@@ -105,15 +104,10 @@ export class Application extends React.Component {
                             <CardTitle component="h2">{_("Directories & files")}</CardTitle>
                             <Flex flexWrap={{ default: 'nowrap' }}>
                                 <SearchInput placeholder={_("Filter directory")} />
-                                <Dropdown toggle={() => {}} />
-                                <Button variant="secondary">
-                                    <Icon>
-                                        <ListIcon />
-                                    </Icon>
-                                </Button>
+                                <DropdownBasic />
```

Comment:
Please rename this to ViewSelector. This is going to be the component to switch from list to grid view.